### PR TITLE
Removed the Completion date from UI

### PR DESCRIPTION
**Pull request by arjund-06**
Signed-off-by: Arjun Dave <davearjun25@gmail.com>

### Functionality:
Completion date is not shown in task details and now is visible in the activity area

### Solution:
Removed the completion date and made changes in the look

### Risk level:
- [ ] high 
- [ ] medium
- [x] low

### How to test:
ng serve
firebase emulators:start
View any task detail page

Pictures:
image

### CODE REVIEW

File: src/app/body/task-details/task-details.component.css [hunk 1]
```diff
@@ -34,7 +34,7 @@
 }
 
 .sprint {
-    color: #5A20F0;
+    color: #10BE65;
```
Comment (viv9k) [inline src/app/body/task-details/task-details.component.css:37]: Is this change needed?

**Review by viv9k — CHANGES_REQUESTED** [review]
Please rectify and fix the required changes.

File: src/app/body/task-details/task-details.component.css (hunk 1 above)
Comment (arjund-06) [reply src/app/body/task-details/task-details.component.css:37]: Yes, you had asked me to change the task ID color, so this is the style for that

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required changes asap

File: src/app/body/task-details/task-details.component.css (hunk 1 above)
Comment (arjund-06) [reply src/app/body/task-details/task-details.component.css:37]: This is the green color used throughout the app

**Review by viv9k — APPROVED** [review]
Ship it
